frontend/packages/integration-tests-cypress/tests/i18n/pseudolocalization.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { checkErrors } from '../../support';
+import { isLoaded } from '../../views/common';
+import { masthead } from '../../views/masthead';
+
+describe('Localization', () => {
+  before(() => {
+    cy.login();
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  it('pseudolocalizes masthead', () => {
+    cy.log('test masthead');
+    cy.visit('/dashboards?pseudolocalization=true&lng=en');
+    masthead.clickMastheadLink('help-dropdown-toggle');
+    cy.byTestID('application-launcher-item', /\[[^a-zA-Z]+\]/);
+  });
+
+  it('pseudolocalizes navigation', () => {
+    cy.log('test navigation');
+    cy.visit('/dashboards?pseudolocalization=true&lng=en');
+    cy.byTestID('nav', /\[[^a-zA-Z]+\]/);
+  });
+
+  it('pseudolocalizes activity card', () => {
+    cy.log('test activity card components');
+    cy.visit('/dashboards?pseudolocalization=true&lng=en');
+    cy.byTestID('activity', /\[[^a-zA-Z]+\]/);
+    cy.byTestID('activity-recent-title', /\[[^a-zA-Z]+\]/);
```

Comment:
Can we create a const for this in `frontend/packages/integration-tests-cypress/views/common.ts`?  Something like `export const pseudoi18nRegex = /\[[^a-zA-Z]+\]/`  -thanks

Reply:
Found a better way to do this, I think, pls see: https://github.com/rebeccaalpert/console/pull/11